### PR TITLE
[CA-1254] Ci billing project left menu

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

File: src/pages/billing/List.js
Comment (s-rubenstein) [inline src/pages/billing/List.js]: @panentheos this is the thing we were hoping to talk to you about. It seems like there should be a better way to make this 'fill up the total height of the screen, except the top bar, and scroll separately from the page' than this, but I couldn't figure it out.

**Review by gpolumbo-broad — APPROVED** [review]

File: src/pages/billing/List.js
Comment (marctalbott) [inline src/pages/billing/List.js]: I can't see the mockup anymore (we hit some quota), but I'm guessing this is part of it? I prefer the plus circle button, but I leave it up to the experts

**Review by marctalbott — COMMENTED** [review]
I'm not exactly sure where this is in the code otherwise I'd put it there, but the gap between the 'New Billing Project' button and the 'Owned by You' section seems too tall to me. I included screenshots of this and the existing to make it easier to compare
Screen Shot 2021-03-25 at 10 53 02 AM
Screen Shot 2021-03-25 at 10 53 11 AM

File: src/pages/billing/List.js
Comment (andy7i) [reply src/pages/billing/List.js]: yeah, exactly:

mock on left, local on right:

Screen Shot 2021-03-25 at 11 08 28 AM

Comment (marctalbott) [reply src/pages/billing/List.js]: 👍

Should 'Owned by You' and 'Shared with You' be bold like in the mock?

File: src/pages/billing/List.js
Comment (andy7i) [inline src/pages/billing/List.js:84]: with fontWeight change:
Screen Shot 2021-03-25 at 11 15 11 AM

**Review by andy7i — APPROVED** [review]
added a fontWeight to match mock. not sure about what testing we need to do, but LGTM otherwise

File: src/pages/billing/List.js
Comment (andy7i) [reply src/pages/billing/List.js]: haha yeah I just added that

**Review by marctalbott — APPROVED** [review]

File: src/pages/billing/List.js
Comment (panentheos) [reply src/pages/billing/List.js]: Hmm, this is a tricky one. There are some assumptions in the top level wrappers that make this difficult. Not comprehensive, but a couple suggestions:
* The `position: fixed` is probably not necessary
* The righthand container also needs to be independently scrollable in this kind of layout
* Be sure to test in the BioDataCatalyst site skin, since that has a persistent footer that will interact with this.

Comment (s-rubenstein) [reply src/pages/billing/List.js]: Good calls, all around! `position: fixed` ended up not necessary, I think the righthand container being scrollable looks and feels better (otherwise it scrolled with the page, leaving the side list behind. The biodatacatalyst skin does have the issue of adding its own scrollbar because of the height issues. I am going to work around this by making a `pageHeightWithoutFooterOrHeader` function that takes that into account.

File: src/pages/billing/List.js
Comment (gpolumbo-broad) [inline src/pages/billing/List.js]: I am not sure which change might have broken it, but the sidebar scrolling is now totally not working for me.  The whole page is scrolling off of the header.

**Review by gpolumbo-broad — CHANGES_REQUESTED** [review]
In addition to this no longer being functional for me, there is one other thing that sort of fell off of our radar.

We had talked about possibly keeping the "New Billing Project" button and left nav heading "locked" at the top.  So in the left nav, when scrolling, you're only scrolling through the "Owned by you" and "Shared with you" lists.  People seemed on board with that idea before, just curious where people stand now.

**Review by gpolumbo-broad — APPROVED** [review]

**Pull request merged**
